expfmt/encode.go

```diff
@@ -36,26 +36,40 @@ func (e encoder) Encode(v *dto.MetricFamily) error {
 	return e(v)
 }
 
-// Negotiate returns the Content-Type based on the given Accept header.
-// If no appropriate accepted type is found, FmtText is returned.
-func Negotiate(h http.Header) Format {
+// Negotiate returns the Content-Type based on the given Accept header.  If no
+// appropriate accepted type is found, FmtText is returned.  Formats can be
```

Comment:
Might want to clarify that FmtText is Prometheus format here, so it's not confused with OM text format. Possibly also rename the constants to include Prom too.

Don't we use a single space after a period usually?

Reply:
I clarified in the comment.

At this point, where the OM support is still work in progress, I'd recommend not to potentially break any users of this package if we can avoid it (even if we technically have the right to do so). I'd rather do one breaking change here when we finalize the support and want to have everything nice nice and tidy. So I'd suggest to keep FmtText for now.

The double space was a hiccup of my Emacs formatting settings.